### ISSUE TITLE
DEPRECATION: Use of `merge` has been deprecated

### ISSUE THREAD

**Issue by raphaelns-developer** [simplabs/ember-cookies#206]
```
DEPRECATION: Use of `merge` has been deprecated. Please use `assign` instead. [deprecation id: ember-polyfills.deprecate-merge] See https://emberjs.com/deprecations/v3.x/#toc_ember-polyfills-deprecate-merge for more details.
```

"ember-source": "~3.7.0",
"ember-cookies": "0.3.1"

**Comment by adambedford** [comment]
+1 Getting this as well on 3.7. Looks like it was deprecated in 3.6

**Comment by marcoow** [comment]
fixed by #204
